static/src/javascripts/projects/common/views/svgs.js

```diff
@@ -107,7 +109,9 @@ define([
         externalLink: externalLink,
         tick: tick,
         glabsLogoSmall: glabsLogoSmall,
-        adblockCoins: adblockCoins,
+        adblockCoinsUK: adblockCoins,
+        adblockCoinsUS: adblockCoinsUS,
+        adblockCoinsINT: adblockCoinsUS, //deliberate
```

Comment:
I think any proprietary SVG mappings for our banner should happen in `adblock-banner` rather than the generic SVG fetch module.